admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -71,17 +71,27 @@
       :selected-files="selectedFiles"
       :dropzone="dropzone"
       @unselectAll="unselectAll"
-      @removeSelection="removeSelection"
+      @removeSelection="toggleModal"
```

Comment:
Isn't it safer to have distinctive `showModal/hideModal` method? Or at least to have a boolean in `toogleModal` Especially since I don't see any management of closing the modal, so the current state might not always be what you expect and you could end up closing it while you wanted to show it (or the opposite)